develop/tutorials/articles/260-themes-and-layout-templates/15-specifying-color-schemes.markdown

```diff
@@ -0,0 +1,110 @@
+# Specifying Color Schemes in your Theme [](id=specifying-color-schemes)
+
+You can provide various "flavors" of your theme by creating color schemes for
+it. Color schemes let you keep the styles and overall design for your theme,
+while at the same time giving a new look for your users to enjoy. You specify
+color schemes with a CSS class name, which of course also lets you choose
+different background images, different border colors, and more.
+ 
+![Figure 1: You can offer eye-pleasing color schemes for your themes.](../../images/theme-color-schemes.png)
+
+Follow these steps to create color schemes for your theme:
+
+1. Create a folder for your color schemes (`color_schemes` for example) in your 
+   theme's `css` folder.
+   
+2. Create a `.scss` file in your color schemes folder for each color scheme 
+   your theme supports. If you don't specify a `.scss` file for a color scheme, 
+   the theme's default color scheme is used.
+
+3. Prefix all your CSS styles with the name of your color scheme. The color 
+   scheme CSS class is placed on the page's `<body>` element, so you can use it 
+   to identify your styling. For example, you would prefix all the styles with 
+   the word *day* in a color scheme CSS file named `_day.scss`: 
+
+        body.day { background-color: #DDF; }
+        .day a { color: #66A; }
+
+    +$$$
+    
+    **Note:** The `Default` color scheme does not require a prefix, as it uses 
+    your theme's `_custom.scss` for styling.
+    
+    $$$
+
+4. Import your color scheme `.scss` files into your `_custom.scss` file. The 
+   example below imports `_day.scss` and `_night.scss` files:
+
+        @import "color_schemes/day";
+        @import "color_schemes/night";
+
+5. Open your theme's `liferay-look-and-feel.xml` file and follow the pattern 
+   below to specify the default color scheme:
+
+        <theme id="my-theme-id" name="My Theme Name">
+           <color-scheme id="01" name="Default">
+               <default-cs>true</default-cs>
+               <css-class>default</css-class>
+               
+               <color-scheme-images-path>
+                   ${images-path}/my_color_schemes_folder_name/${css-class}
+               </color-scheme-images-path>
+           </color-scheme
+           ...
+        </theme>
+
+    Specifying a default color scheme lets you return to the default look and 
+    feel for your theme.
+    
+    Note that color schemes are sorted alphabetically by `name` rather than `id`. 
+    For example, a color scheme named `Day` and `id` `02` would be selected by 
+    default over the `Default` color scheme with `id` `01`. The `<default-cs>` 
+    element overrides the alphabetical sorting and sets the color scheme that is 
+    selected by default, when the theme is chosen. Adding this element to the 
+    default color scheme ensures that it is selected when the theme is chosen.
```

Comment:
Yeah, it's unfortunate wording. The default color scheme (the original color scheme) is named 'default.' I don't believe you can change that. the original color scheme isn't selected by default though, if you have another color scheme that comes before it alphabetically, so you have to set that element to true to ensure that it is. Sounds like a bug.